doc/developer_guide/dependency.rst

```diff
@@ -590,14 +590,22 @@ Dependency Tools
 ================
 
 PSyclone contains a class that builds upon the data-dependency functionality
-to provide useful tools for dependency analaysis. It especially provides
+to provide useful tools for dependency analysis. It especially provides
 messages for the user to indicate why parallelisation was not possible. It
 uses `SymPy` internally to compare expressions symbolically.
 
 .. autoclass:: psyclone.psyir.tools.dependency_tools.DependencyTools
     :members:
 
 
+.. note:: PSyclone provides :ref:`user_guide:replace_induction_variable_trans`,
```

Comment:
I believe this is because the label in the user guide is not on read-the-docs. When I grep'ed, the new label is defined in the users guide. Or is there anything I can do before this is merged?

Reply:
If the Sphinx link checking is passing in the test suite then this is OK.